pkgs/top-level/all-packages.nix

```diff
@@ -5855,14 +5838,7 @@ with pkgs;
   xbursttools = callPackage ../tools/misc/xburst-tools {
     # It needs a cross compiler for mipsel to build the firmware it will
     # load into the Ben Nanonote
-    gccCross =
-      let
-        pkgsCross = nixpkgsFun {
-          # Ben Nanonote system
-          crossSystem = lib.systems.examples.ben-nanonote;
-        };
-      in
-        pkgsCross.buildPackages.gccCrossStageStatic;
+    gccCross = pkgsCross.ben-nanonote.buildPackages.gcc;
```

Comment:
Should this be gcc or gccCrossStageStatic? @Ericson2314

Reply:
`gccCrossStageStatic`. No `libc` is needed.